client/src/pages/ShoppingCartPage.js

```diff
@@ -120,41 +126,70 @@ function ShoppingCartPage(props) {
       productID,
       quantity,
     }));
+
+    let shipmentInfo = {
+      date: deliveryDate,
+      time: deliveryTime,
+      address: deliveryAddress,
+      fee: 0
+    };
+
+    if(deliveryType === "Shipment")
+      shipmentInfo.fee = 20;
```

Comment:
Add a comment to specify that this solution is just temporary, and it need to be modified in the future